src/fsharp/FSharp.Core/async.fsi

```diff
@@ -1072,7 +1072,16 @@ namespace Microsoft.FSharp.Control
             /// <summary>Returns an asynchronous computation that, when run, will wait for a response to the given WebRequest.</summary>
             /// <returns>An asynchronous computation that waits for response to the <c>WebRequest</c>.</returns>
             /// 
-            /// <example-tbd></example-tbd>
+            /// <example id="get-response">
+            /// <code lang="fsharp">
+            /// let responseStreamToString = fun (responseStream : System.Net.WebResponse) ->
```

Comment:
You can add `open System` and `open System.IO` and `open System.Net` at the top of the sample I think.  Best to keep the code crisp.